_layouts/contact.html

```diff
@@ -0,0 +1,11 @@
+---
+layout: page
+---
+
+<article class="page">
+
+  <h2>{{ page.title }}</h2>
+
+  <div class="entry abouts {{page.wrapperclass}}">{{ content }}</div>
```

Comment:
no need for the abouts or {{page.wrapperclass} classes (since it doesn't seem like you're specifying a wrapperclass value in the template.